internal/apps/dop/endpoints/iteration.go

```diff
@@ -235,6 +237,27 @@ func (e *Endpoints) DeleteIteration(ctx context.Context, r *http.Request, vars m
 		return errorresp.ErrResp(err)
 	}
 
+	itemIDs, err := e.issueService.DBClient().GetIssuesIDByIterationID(iterationID)
+	if err != nil {
+		return apierrors.ErrDeleteIteration.InternalError(err).ToResp(), nil
+	}
+
+	if len(itemIDs) > 0 {
+		ctx = apis.WithUserIDContext(ctx, identityInfo.UserID)
+		if onlyIteration == "true" {
+			err = e.issueService.BatchUpdateIssueIterationIDByIterationID(ctx, iterationID, -1)
+			if err != nil {
+				return apierrors.ErrDeleteIteration.InternalError(err).ToResp(), nil
+			}
+			goto come
+		} else if onlyIteration == "false" {
```

Comment:
no need else if, all non-true treat as `false`.

Reply:
and no need `goto come`

Reply:
OK，thanks